lib/presentation/home_screen/home_screen.dart

```diff
@@ -21,27 +21,30 @@ class _HomeScreenState extends State<HomeScreen> {
     mapController = controller;
   }
 
-
   @override
   Widget build(BuildContext context) {
-
     return Scaffold(
       appBar: AppBar(
-        title: Text(widget.title),
+        title: Text(AppLocalizations.of(context)!.appName),
       ),
-      drawer: MyDrawer(
+      drawer: const MyDrawer(
         userName: 'Help A Paw',
         userImage: 'assets/images/paww.png',
       ),
-      body: GoogleMap(
-        onMapCreated: _onMapCreated,
-        initialCameraPosition: CameraPosition(
-          target: _center,
-          zoom: 11.0,
-        ),
-        zoomControlsEnabled: false,
-        myLocationEnabled: true,
+      body: Stack(
```

Comment:
We can also probably get rid of the Stack if it will contain a single child!?

Reply:
Okay On It